index.js

```diff
@@ -48,14 +49,18 @@ module.exports = async function(robot) {
   // Kredits contracts setup
   //
 
+  const opts = { ipfsConfig };
+
+  if (daoAddress) {
+    opts.addresses = { Kernel: daoAddress };
+  } else {
+    opts.apm = 'open.aragonpm.eth';
```

Comment:
we currently default already to `open.aragonpm.eth` https://github.com/67P/kredits-contracts/blob/master/lib/contracts/kernel.js#L9 (and: https://github.com/67P/kredits-contracts/blob/master/arapp.json#L37) 

so we should not need to set this here. 
With the new aragon CLI version we also should be able to default to `open.aragonpm.eth` in all environments and no longer need to set this manually. 

I'd either make this also configurable through an env variable or just ignore it let kredits-contracts use the default.

Reply:
Thx.